### ISSUE TITLE
gitpod/workspace-mysql no longer installs apache2

### ISSUE THREAD

**Issue by apolopena** [gitpod-io/workspace-images#675]
### Bug description

Any `.gitpod.Dockerfile` using:
```yml
FROM gitpod/workspace-mysql
```
or 
```yml
FROM gitpod/workspace-mysql:latest
```
no longer has apache2 installed.

This breaks alot of things. 

All docker image build steps will break if an assumed directory such as /etc/apache2/ is used to lets say copy a `apache2.conf ` file over or something like that which is a typical thing to do.

Note: `workspace-full` is unaffected.

### Steps to reproduce

Create a workspace from the url below which is a copy of https://github.com/gitpod-io/apache-example
with one single change: https://github.com/apolopena/gitpod-bug-8113/commit/b6148ea255e817f580b1c282f71d7db08ae809e3

https://gitpod.io/#/https://github.com/apolopena/gitpod-bug-8113

Terminal output: `bash: apachectl: command not found`
Running: `apache2 -v` gives the error:
```log
bash: apache2: command not found
```
Apache2 is not installed.

### Workspace affected

Any workspace that uses gitpod/workspace-mysql

### Expected behavior

Any image built on `workspace-full` such as `workspace-mysql` should have apache2. Removing this is not backwards compatible.

### Example repository

https://github.com/apolopena/gitpod-bug-8113

### Anything else?

In order to maintain vital backwards compatibility in the gitpod workspace images, things like this should not change.

**Comment by apolopena** [comment]
@jimmybrancaccio
Maybe you can shed some light on this? Some workaround that doesn't involved implementing my own mysql integration on top of workspace-full?
https://github.com/apolopena/gitpod-laravel-starter/issues/200#issuecomment-1034165012

**Comment by jimmybrancaccio** [comment]
@apolopena

> Unfortunately legacy-dazzle-v1 is buggy with ca certificates and should not be used.

I wouldn't say it's buggy, as there's no bug. However you do need to manually update the packages as I noted in #599.

> So now it seems that I should probably maintain my own image and work Gitpods changes in as I see fit.

You can definitely do this as well, the joys of open source!

I will probably end up having to build my own images as well since I build with Laravel and need MySQL and Apache too.

**Comment by princerachit** [comment]
Thanks for pointing this out. I am working internally with the team on this and would update you soon.

**Comment by davekiss** [comment]
I'm also experience an issue in the mongo image that may be related. Trying to set my version of node to something other than the default (following the approach recommended in the docs here: https://www.gitpod.io/docs/languages/javascript#node-versions)

This used to work fine until a recent update to the Dockerfile, now I get this:

```
#6 [3/5] RUN bash -c 'VERSION="14.18.3"     && source $HOME/.nvm/nvm.sh && nvm install $VERSION     && nvm use $VERSION && nvm alias default $VERSION'
#6 0.287 bash: /home/gitpod/.nvm/nvm.sh: No such file or directory
#6 ERROR: process "/bin/sh -c bash -c 'VERSION=\"14.18.3\"     && source $HOME/.nvm/nvm.sh && nvm install $VERSION     && nvm use $VERSION && nvm alias default $VERSION'" did not complete successfully: exit code: 1
------
 > [3/5] RUN bash -c 'VERSION="14.18.3"     && source $HOME/.nvm/nvm.sh && nvm install $VERSION     && nvm use $VERSION && nvm alias default $VERSION':
#6 0.287 bash: /home/gitpod/.nvm/nvm.sh: No such file or directory
------
```

Let me know if you'd rather have a new issue or if this will be corrected in #677

**Comment by kylos101** [comment]
> I'm also experiencing an issue in the mongo image that may be related. Trying to set my version of node to something other than the default (following the approach recommended in the docs here: https://www.gitpod.io/docs/languages/javascript#node-versions)
> 
> This used to work fine until a recent update to the Dockerfile, now I get this:
> 
> ```
> #6 [3/5] RUN bash -c 'VERSION="14.18.3"     && source $HOME/.nvm/nvm.sh && nvm install $VERSION     && nvm use $VERSION && nvm alias default $VERSION'
> #6 0.287 bash: /home/gitpod/.nvm/nvm.sh: No such file or directory
> #6 ERROR: process "/bin/sh -c bash -c 'VERSION=\"14.18.3\"     && source $HOME/.nvm/nvm.sh && nvm install $VERSION     && nvm use $VERSION && nvm alias default $VERSION'" did not complete successfully: exit code: 1
> ------
>  > [3/5] RUN bash -c 'VERSION="14.18.3"     && source $HOME/.nvm/nvm.sh && nvm install $VERSION     && nvm use $VERSION && nvm alias default $VERSION':
> #6 0.287 bash: /home/gitpod/.nvm/nvm.sh: No such file or directory
> ------
> ```
> 
> Let me know if you'd rather have a new issue or if this will be corrected in #677

@davekiss  That should be resolved in #677, as we changed the base image for mongo from base to full (which means it'll include node again). The latest tag should be updated in roughly 1 hour from now.

**Comment by apolopena** [comment]
@kylos Can please update mysql from base to full as well?
